apps/entity-service/src/entities/entities.service.ts

```diff
@@ -68,6 +80,17 @@ const ASSOCIATION_PROCESSORS = {
       attributes: ["uuid", "name", "taxonId", "collection", [fn("SUM", col("amount")), "amount"]],
       group: ["taxonId", "name", "collection"]
     })
+  ),
+  disturbances: AssociationProcessor.buildSimpleProcessor(
+    DisturbanceDto,
+    ({ id: disturbanceableId }, disturbanceableType) =>
+      Disturbance.findAll({ where: { disturbanceableType, disturbanceableId, hidden: false } })
+  ),
+  invasives: AssociationProcessor.buildSimpleProcessor(InvasiveDto, ({ id: invasiveableId }, invasiveableType) =>
+    Invasive.findAll({ where: { invasiveableType, invasiveableId, hidden: false } })
+  ),
+  stratas: AssociationProcessor.buildSimpleProcessor(StrataDto, ({ id: stratasableId }, stratasableType) =>
+    Strata.findAll({ where: { stratasableType, stratasableId, hidden: false } })
```

Comment:
Since these are all explicitly only returning non-hidden entries, it seems like we should be able to drop `hidden` from the DTOs as well.